client/init.c

```diff
@@ -205,75 +207,74 @@ TDNFRefreshSack(
         pTdnf->pArgs->nRefresh = 1;
     }
 
-    dwError = TDNFInitCmdLineRepo(pTdnf, pSack);
-    BAIL_ON_TDNF_ERROR(dwError);
-
-    /* First repo is the "@cmdline" repo, which always exists.
-     * skip over it - options do not apply, and it is initialized. */
-    if(pTdnf->pRepos->pNext)
+    for(pRepo = pTdnf->pRepos; pRepo; pRepo = pRepo->pNext)
     {
-        PTDNF_REPO_DATA_INTERNAL pTempRepo = pTdnf->pRepos->pNext;
-        while(pTempRepo)
+        /* skip the @cmdline repo - options do not apply, and it is
+           initialized. */
+        if (strcmp(pRepo->pszName, "@cmdline") == 0)
+        {
+            continue;
+        }
+
+        if(!pRepo->nEnabled)
```

Comment:
nit: the above two if conditions can be combined into one

Reply:
This was intended because I thought it's easier to read, but I think it's fine when the two conditions are on separare lines.